.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [6.x, 8.x, 10.x, 12.x, 13.x]
```

Comment:
The minimum version supported by pino@6 is node 10.

Reply:
Good point. I blindly copied the travis file. Removed now.